openlibrary/templates/books/edit/edition.html

```diff
@@ -613,7 +613,7 @@
                     <span class="tip"></span>
                 </div>
                 <div class="input">
-                    $ weight = book.get_weight() or storage(value="", units="grams")
+                    $ weight = book.get_weight() or storage(value="", units="kilos")
```

Comment:
The default weight should perhaps be grams; that's what other sites use and a more reasonable default than kg for books.

```suggestion
                    $ weight = book.get_weight() or storage(value="", units="grams")
```